docs/source/developer/extension_tutorial.rst

```diff
@@ -239,7 +239,7 @@ definition so that it reads like so:
      * Initialization data for the jupyterlab_apod extension.
      */
     const extension: JupyterFrontEndPlugin<void> = {
-      id: 'jupyterlab_apod',
+      id: 'jupyterlab-apod',
```

Comment:
This change is just because that's what is generated when running the Cookiecutter, it's also the name used in the generated `package.json`.

Otherwise, the tutorial would be asking the user to change the generated `jupyterlab-apod` to `jupyterlab_apod`.